pubspec.yaml

```diff
@@ -11,7 +11,7 @@ dependencies:
   cli_completion: ^0.2.0
   glob: ^2.0.2
   lcov_parser: ^0.1.2
-  mason: ">=0.1.0-dev.41 <0.1.0-dev.42"
+  mason: 0.1.0-dev.46
```

Comment:
Was it intentional to remove the range?

Reply:
yeah, since dev releases sometimes carry breaking changes, there is not much value to add a range there (also because now the range would be >=46 <47)

Reply:
Ah u didn't add the `^`.  The reason we had the range is because of the breaking changes, if someone installed the CLI while mason broke something and the installation installed that new version then the CLI would also be broken.

What was the reason for using a range instead of doing it like Renan did @felangel ?

Reply:
The range would allow for newer versions without always having to manually bump this version. I'm hoping we can release a stable 0.1.0 soon and there shouldn't really be breaking changes at this point but I think it's also fine locking the version if you really want to be extra cautious -- it's just putting a bit more effort on maintainers to keep this value up-to-date and also restricts the number of folks who can use the brick. For example, if mason 0.1.0-dev.47 comes out then folks who are using the newer version of mason won't be able to use this brick until we upgrade the constraint and people who are using older versions 0.1.0-dev.45 etc. also won't be able to use this brick unless they upgrade.

Reply:
This PR no longer changes the mason dependency directly